src/tenant/components/contact/Contact.css

```diff
@@ -1,6 +1,6 @@
 
   
-  form{
```

Comment:
Please be specific in this class names to avoid conflict with other form styles.